apps/src/aichat/views/modelCustomization/PromptCustomization.tsx

```diff
@@ -35,6 +43,33 @@ const PromptCustomization: React.FunctionComponent = () => {
   return (
     <div className={styles.verticalFlexContainer}>
       <div>
+        <div className={classNames(styles.inputContainer)}>
+          <SimpleDropdown
+            labelText="Pick a model:"
+            onChange={e => setChosenModel(e.target.value)}
+            items={[
+              {value: 'llama2', text: 'LLama 2'},
+              {value: 'gpt', text: 'ChatGPT'},
+            ]}
+            selectedValue={chosenModel}
+            name="model"
+            size="s"
+            className={styles.updateButton}
+          />
+          <div>
```

Comment:
Do we need this div?

Reply:
I don't believe so! Removing. Wish there was an "extra div" linter 🤦